File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// OldestPeopleRecords SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class OldestPeopleRecordsMakeContext
{
    public static function call(array $ctxmap, ?OldestPeopleRecordsContext $basectx): OldestPeopleRecordsContext
    {
        return new OldestPeopleRecordsContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# OldestPeopleRecords SDK utility: make_context
require_relative '../core/context'
module OldestPeopleRecordsUtilities
  MakeContext = ->(ctxmap, basectx) {
    OldestPeopleRecordsContext.new(ctxmap, basectx)
  }
end
